Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -54,14 +55,28 @@ class JSWebAssemblyArray final : public JSNonFinalObject {
         return Structure::create(vm, globalObject, prototype, TypeInfo(ObjectType, StructureFlags), info());
     }
 
-    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::Type elementType, size_t size, FixedVector<uint32_t>&& payload)
+    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, FixedVector<uint32_t>&& payload)
     {
         JSWebAssemblyArray* array = new (NotNull, allocateCell<JSWebAssemblyArray>(vm)) JSWebAssemblyArray(vm, structure, elementType, size, WTFMove(payload));
         array->finishCreation(vm);
         return array;
     }
 
-    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::Type elementType, size_t size, FixedVector<uint64_t>&& payload)
+    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, FixedVector<uint64_t>&& payload)
+    {
+        JSWebAssemblyArray* array = new (NotNull, allocateCell<JSWebAssemblyArray>(vm)) JSWebAssemblyArray(vm, structure, elementType, size, WTFMove(payload));
+        array->finishCreation(vm);
+        return array;
+    }
+
+    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, FixedVector<uint16_t>&& payload)
+    {
+        JSWebAssemblyArray* array = new (NotNull, allocateCell<JSWebAssemblyArray>(vm)) JSWebAssemblyArray(vm, structure, elementType, size, WTFMove(payload));
+        array->finishCreation(vm);
+        return array;
+    }
+
+    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, FixedVector<uint8_t>&& payload)
```

Comment:
Minor nit: can we order the constructors the same as the union? (increasing order of type width)